mantidproject/mantid
# Close multiple windows

Description of work.

**To test:**

1. Start Mantidplot
1. Create at least 2 of each type of window (look in the File New menu, and then add the workspace matrix view)
1. Select a window and use Windows->Close All ??? Windows
1. that menu option should only appear if you have more than 1 of that type of window.

Fixes #18631.

### RELEASE NOTES

docs/source/release/ui.rst

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
